.github/workflows/image-push-master.yml

```diff
@@ -1,4 +1,8 @@
 name: "Push images on merge to master"
+
+env:
+  BUILD_PLATFORM: linux/amd64,linux/arm64,linux/ppc64le
```

Comment:
please make this list equal (or the other way around) with the matrix of the test

Reply:
OKAY,  Thanks for your comments.

I'm not opposed to using lists. but I wonder why `list` is needed here instead of a comma-separated string ? docker buildx platforms as input parameters like this.

Reply:
Well, I agree with you, I made some changes, can you take a look? thanks